src/components/DigitalAnalytics.astro

```diff
@@ -0,0 +1,13 @@
+---
+interface Props {
+  dapAgencyCode: string;
+  dapSubAgencyCode?: string | null;
+}
+
+const { dapAgencyCode, dapSubAgencyCode } = Astro.props;
+let dapSrc = `https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js?agency=${dapAgencyCode}`;
+
+dapSubAgencyCode ? (dapSrc += `&subagency=${dapSubAgencyCode}`) : null;
+---
+
+<script id="_fed_an_ua_tag" src={dapSrc}></script>
```

Comment:
I think this needs to be set to `is:inline` https://docs.astro.build/en/guides/client-side-scripts/#load-external-scripts